tests/src/main/java/com/hazelcast/stabilizer/tests/helpers/HazelcastTestUtils.java

```diff
@@ -167,16 +167,32 @@ public static HazelcastInstanceImpl getHazelcastInstanceImpl(HazelcastInstance h
         return impl;
     }
 
-    public static long nextKeyOwnedBy(long key, HazelcastInstance instance) {
-        final Member localMember = instance.getCluster().getLocalMember();
-        final PartitionService partitionService = instance.getPartitionService();
-        for (; ; ) {
+    /**
+     * Returns the next {@code long} key owned by the given Hazelcast instance.
+     * <p/>
+     * If no key is found after 100 retries a {@link RuntimeException} is thrown.
+     *
+     * @param instance  Hazelcast instance to search next key for
+     * @param lastKey   last key to start search from
+     * @param increment increment to add if next key did not belong to given instance
+     * @return next key owned by given Hazelcast instance
+     * @throws RuntimeException if no key could be found after 100 retries
+     */
+    public static long nextKeyOwnedBy(HazelcastInstance instance, long lastKey, int increment) {
```

Comment:
we we really need to complicate this method with an increment,  which has not real meaning.   we just want to return the next long that is owned by the instance,  (inclusive of LastKey value)